docs/useful_tools.md

```diff
@@ -88,12 +88,15 @@ We provide `tools/ort_test.py` to evaluate ONNX model with ONNXRuntime backend.
   pip install onnx onnxruntime-gpu
   ```
 
+- Install TensorRT python package(optional)
```

Comment:
could add the doc link here for reference: https://mmcv.readthedocs.io/en/latest/tensorrt_plugin.html#how-to-build-tensorrt-plugins-in-mmcv